README.md

```diff
@@ -109,14 +113,39 @@ overrides: {
 }
 ```
 
+## ERL (Elevated Rate Limits)
+ERL is a feature that allows you to define a different set of limits that kick in when the bucket is empty.
+The feature aims to provide a way to temporarily allow a higher rate of requests when the bucket is empty, for a limited period of time.
+
+To be able to allow its use within limitd-redis, you need to:
+1. call the `take` method with the parameter `allowERL` set to `true`.
+2. pass the `erlIsActiveKey` parameter with the identifier of the ERL activation for the bucket. This works similarly to the `key` you pass to `limitd.take`, which is the identifier of the bucket; however it's used to track the ERL activation for the bucket instead.
+3. make sure that the bucket definition has ERL configured.
+
+You can configure elevated limits inside your bucket definitions as follows:
+
+```js
+buckets = {
+  ip: {
+    size: 10,
+    per_second: 5,
+    elevated_limits: {
+      size: 100, // new bucket size. already used tokens will be deducted from current bucket content upon ERL activation.
+      per_second: 50, // new bucket refill rate. You can use all the other refill rate configurations defined above, such as per_minute, per_hour, per_interval etc.
+      erl_activation_period_seconds: 300, // for how long the ERL configuration should remain active once activated.
+    }
+  }
+}
+```
+
 ## Breaking changes from `Limitdb`
 
 * Elements will have a default TTL of a week unless specified otherwise.
 
 ## TAKE
 
 ```js
-limitd.take(type, key, [count], (err, result) => {
+limitd.take(type, key, [count], erlActiveKey, allowERL (err, result) => {
```

Comment:
are `erlActiveKey` and `allowERL` mandatory? I thought they were optional (reading from description of the fields below).

Does it make sense that only one of those is provided? maybe we can say that if `erlActiveKey` is passed then `allowERL` is implied?

In any case, missing a comma between `allowERL` and the function.

```suggestion
limitd.take(type, key, [count], [erlActiveKey], [allowERL], (err, result) => {
```

Reply:
Are these the real parameters `take` take? In `lib/client.js` (which I'm not really sure if that is `limitd.take(...)` we have:

```
take(type, key, opts, cb) { ... }
```

So maybe it should be written as

```suggestion
limitd.take(type, key, { [count], [erlActiveKey], [allowERL]}, (err, result) => {
```

?

Reply:
That's likely. Also found an [example](https://github.com/auth0/api2/blob/bdabb220e96af8e03965f0d20a0b95ea51a25b0d/api/anomaly/blocks/ips/lib/anomaly-detection.js#L42) that passes `configOverride` together with `count`.  Updated accordingly.
```
anomalyDetectionLimitrClient.take(bucketName, key, { configOverride, count: 0 }, (err, response)
```
---

> are erlActiveKey and allowERL mandatory? I thought they were optional

`allowERL` is optional. `erlIsActiveKey` is mandatory ONLY IF you are passing a bucket that has an `elevated_limits` section in its definition.